pkg/protocol/sybilprotection/sybilprotectionv1/performance/rewards.go

```diff
@@ -61,23 +50,64 @@ func (t *Tracker) ValidatorReward(validatorID iotago.AccountID, stakeAmount iota
 		if err != nil {
 			return 0, 0, 0, ierrors.Wrapf(err, "failed to get pool stats for epoch %d and validator accountID %s", epochIndex, validatorID)
 		}
+
 		if poolStats == nil {
 			return 0, 0, 0, ierrors.Errorf("pool stats for epoch %d and validator accountID %s are nil", epochIndex, validatorID)
 		}
 
-		unDecayedEpochRewards := uint64(rewardsForAccountInEpoch.FixedCost) +
-			decreaseAccuracy(poolStats.ProfitMargin*uint64(rewardsForAccountInEpoch.PoolRewards), profitMarginExponent) +
-			decreaseAccuracy(increasedAccuracyComplement(poolStats.ProfitMargin, profitMarginExponent)*uint64(rewardsForAccountInEpoch.PoolRewards), profitMarginExponent)*
-				uint64(stakeAmount)/
-				uint64(rewardsForAccountInEpoch.PoolStake)
+		// all rewards go to delegators, as the fixed cost was too high
```

Comment:
I don't understand this comment. isn't the fixed cost for the validator meaning that `no` rewards go to the delegators?

Reply:
this is a form of punishing the validator for adding an unreasonably high fixed cost. if fixed cost>pool reward, we "slash" the validator reward

Reply:
however, the way it is now, the validator will get zero for the whole time period, not for the specific epochs where this happened. It's not an urgent matter, but I feel like this can be a problem in the future with the random committees

Reply:
since the validator won't know in advance the order of magnitude of his reward for every single epoch, I think this could become a punishment for unlucky people in the future :/